cfpb/consumerfinance.gov
# Fix aria role on expandable icons

Lighthouse reports that the icons that have `aria-label`s need the proper aria role. This PR adds a `role="img"` on the icons.


## Changes

- Add role="img" on expandable icons.


## How to test this PR

1.  Visit localhost:8000/about-us/blog/ and run lighthouse in the browser dev tools. Compare to production. Accessibility score should be higher.
